Packs/CommonScripts/Scripts/FindSimilarIncidentsV2/FindSimilarIncidentsV2.py

```diff
@@ -117,6 +117,7 @@ def get_incidents_by_keys(similar_incident_keys, time_field, incident_time, inci
                                    .replace('"', r'\"')
                                    .replace("\n", "\\n")
                                    .replace("\r", "\\r")
+                                   if type(t[1]) not in (int, float, bool) else t[1]
```

Comment:
I don't think is that critical but I would do the opposite condition
means if `isinstace(t[1],` basestring) else t[1]`